src/styles/rhd-theme/components/_all-products-listing.scss

```diff
@@ -0,0 +1,42 @@
+.rhd-c-all-products-list {
+  @extend .pf-u-pt-lg;
+  @extend .pf-u-pb-lg;
+
+  h2, h2.pf-c-title {
+    margin-bottom: var(--pf-global--spacer--lg); // 24px
+  }
+
+  ul {
+    @extend .pf-l-grid;
```

Comment:
I would extend `.pf-c-list`, in order to utilize the PatternFly list styles, while still keeping your `list-style: none;` property.